pinc/image_check.inc

```diff
@@ -12,3 +13,14 @@ function get_image_size_error($image_size)
     }
     return $result;
 }
+
+function get_image_size_small_error($image_dimensions)
```

Comment:
I'd recommend this function take `$width` and `$height`, that way it isn't implicitly relying on the output of `getimagesize()`. This way we keep the opaqueness of the data structure immediately adjacent to where it comes from.

Also consider renaming this `get_image_small_dimension_error()` since "size" is used as file size elsewhere in this function. Ditto `$page_image_minimum_size` as `$page_image_minimum_dimension`.

Reply:
Great suggestions.